src/scrapers/base-scraper-with-browser.ts

```diff
@@ -36,7 +36,7 @@ export type PossibleLoginResults = {
 
 export interface LoginOptions {
   loginUrl: string;
-  checkReadiness?: () => Promise<void>;
+  checkReadiness?: () => Promise<unknown>;
```

Comment:
I think if you force me to return a `void`, I can't use a **one-line arrow function** because it returning the expression return type.